roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -0,0 +1,85 @@
+---
+- name: Copy Configuration to temporary conf
+  command: >
+    cp {{ openshift.common.config_base }}/master/admin.kubeconfig /tmp/manageiq_admin.kubeconfig
+
+- name: Add Managment Infrastructure project
+  command: > 
+    {{ openshift.common.admin_binary }} new-project
+    management-infra
+    --description="Management Infrastructure"
+    --config=/tmp/manageiq_admin.kubeconfig
+  register: osmiq_create_mi_project
+  failed_when: "'already exists' not in osmiq_create_mi_project.stderr and osmiq_create_mi_project.rc != 0"
+  changed_when: false
```

Comment:
`changed_when: osmiq_create_mi_project.rc == 0` is probably more appropriate here.